cmd/argo/commands/common/get.go

```diff
@@ -479,7 +479,7 @@ func printNode(w *tabwriter.Writer, node wfv1.NodeStatus, wfName, nodePrefix str
 	var args []interface{}
 	duration := humanize.RelativeDurationShort(node.StartedAt.Time, node.FinishedAt.Time)
 	if node.Type == wfv1.NodeTypePod {
-		podName := util.PodName(wfName, nodeName, templateName, node.ID, podNameVersion)
+		podName := util.PodName(wfName, node.Name, templateName, node.ID, podNameVersion)
```

Comment:
This is great. Do you think we should add a comment in here to explain the difference between "nodeName" and "node.Name"?